src/index.html

```diff
@@ -11,8 +11,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="container"
+      data-qa="card"
+    >
+      <div class="container__img">
+        <img
+          src="images/imac.jpeg"
+          alt="IMac image"
+          class="container__img-imac"
+        />
+      </div>
+
+      <div class="container__texts-inform">
+        <h1 class="container__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
+        <span class="container__span">Product code: 195434</span>
+        <div class="container__stars-reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="container__review-text">Reviews: 5</span>
+        </div>
+        <div class="container__price-container">
+          <div class="price-container price-container--label">Price:</div>
+          <div class="price-container price-container--price">$2,199</div>
+        </div>
+      </div>
+
+      <a
+        class="container__button"
+        data-qa="hover"
+      >
```

Comment:
The <a> tag should have an href attribute to be a valid link. If it's meant to be a button, consider using a <button> element instead.